### PR TITLE
SOAPec

### PR DESCRIPTION
**Pull request by jerowe**
* [ ] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by jerowe** [comment]
```
/anaconda/conda-bld/test-tmp_dir/run_test.sh: line 3: /anaconda/envs/_test/bin/Corrector_AR: cannot execute binary file
```

File: recipes/soapec/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+build:
+  number: 0
+
+package:
+  name: soapec
+  version: '2.01'
+
+requirements:
+  run:
+      - libgcc
```
Comment (bgruening) [inline recipes/soapec/meta.yaml]: You are not compiling.

**Comment by bgruening** [comment]
No ideas about the error. Maybe the binary was compiled on a different architecture?

**Comment by jerowe** [comment]
You're right! Its a precompiled binary. Of course it won't work on osx. Bah!

I'll email the soap people to ask about source code for this and soapdenovo2-gapcloser.

**Comment by bgruening** [comment]
Fantastic, pushing the OpenSource spirit from bioconda down to upstream :)

**Comment by jerowe** [comment]
I don't need to ask. Its right on sourceforge. ;-)

**Comment by jerowe** [comment]
On the osx build:

```
main.cpp:82:31: error: use of undeclared identifier 'PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP'

pthread_mutex_t Block_mutex = PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP;

                              ^

main.cpp:83:30: error: use of undeclared identifier 'PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP'

pthread_mutex_t Kmer_mutex = PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP;
```

Any ideas for me?

**Comment by jerowe** [comment]
I got nothing. Ideas?

**Comment by jerowe** [comment]
Is it alright if I merge the linux build and leave the osx for a later date? I have a few more packages I need to build this week.

**Comment by bgruening** [comment]
This is ok!

**Pull request merged**